triggers/service/src/test/scala/com/digitalasset/daml/lf/engine/trigger/TriggerServiceFixture.scala

```diff
@@ -67,8 +71,62 @@ object TriggerServiceFixture {
     val host = InetAddress.getLoopbackAddress
     val isWindows: Boolean = sys.props("os.name").toLowerCase.contains("windows")
 
-    // Launch a toxiproxy instance. Wait on it to be ready to accept
-    // connections.
+    // Set up an authentication service if enabled
+    val authServiceAdminLedger: Future[Option[(SandboxServer, Port)]] =
```

Comment:
This looks a bit confusing. You now have two ledger futures. Can we only have `ledgerF` and switch which sandbox config we use in there?

Reply:
The auth service uses its own ledger for managing users and service accounts etc, which is the admin ledger here. That's why there are two ledger futures. It should be possible to use the main ledger which the triggers run on as the admin ledger, though I'm not sure there's a real benefit besides saving one sandbox instance.

Reply:
Speeding up tests and probably also simplifying the code seems like a win win to me but I’m fine with leaving that for a separate PR if you prefer.